generator/internal/sidekick/templatedata.go

```diff
@@ -481,3 +246,52 @@ func mapSlice[T, R any](s []T, f func(T) R) []R {
 	}
 	return r
 }
+
+func newOneOf(oneOf *api.OneOf, c api.LanguageCodec, state *api.APIState) *OneOf {
+	return &OneOf{
+		NameToPascal:          c.ToPascal(oneOf.Name),
+		NameToSnake:           c.ToSnake(oneOf.Name),
+		NameToSnakeNoMangling: c.ToSnakeNoMangling(oneOf.Name),
+		FieldType:             c.OneOfType(oneOf, state),
+		DocLines:              c.FormatDocComments(oneOf.Documentation),
+		Fields: mapSlice(oneOf.Fields, func(field *api.Field) *Field {
+			return newField(field, c, state)
+		}),
+	}
+}
+
+// Constructor function for c.Field
```

Comment:
Why comments in some of these functions but not the others?

Reply:
stray comments, removed.